.commitlintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "rules": {
+      "type-enum": [1, "always", [
+      "build",
+      "ci",
+      "docs",
+      "feat",
+      "fix",
+      "perf",
+      "refactor",
+      "revert",
+      "style",
+      "test"
+    ]],
```

Comment:
The terms are overlapping, e.g. what to use when refactor the code which improves performance? What if you fix the doc? What would you do when working on process descriptions, is that doc? What would you choose when removing a function? Is that refactor?

Personally, I would not limit this to a certain set of term, but recommend some terms, e.g. start of "Revert" if you revert a comment.

Btw, starting a commit message with lower case looks strange to my eyes.